apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Editors/index.tsx

```diff
@@ -25,7 +25,7 @@ export function Editors({
   const { metadata } = useMetadata()
   const { value, updateDocumentContent, isSaved } = useDocumentValue()
   const { diff: latteDiff } = useLatteDiff()
-  const { diff: experimentDiff, setDiff: setEditorDiff } = useExperimentDiff()
+  const [experimentDiff, setEditorDiff] = useState<DiffOptions | undefined>()
```

Comment:
`useExperimentDiff` was there to set initial value from the backend. I think we were not doing that so I removed the provider and the hook